_includes/links.md

```diff
@@ -31,8 +31,11 @@
 [mit-license]: https://opensource.org/licenses/mit-license.html
 [morea]: https://morea-framework.github.io/
 [numfocus]: https://numfocus.org/
+[numpy]: https://www.numpy.org/
 [osi]: https://opensource.org
 [pandoc]: https://pandoc.org/
+[pandas]: https://pandas.pydata.org
+[pandas-docs]: https://pandas.pydata.org/pandas-docs/stable/
```

Comment:
Adding links here might cause a problem when syncing with carpentries/styles.
So, let's add them to the files that use them. Later, we can argue whether they should end up in `links.md` that is shared by all lessons.